diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/core/Constants.h b/node_modules/react-native-audio-api/common/cpp/audioapi/core/Constants.h
index 4d82dca..f0121e9 100644
--- a/node_modules/react-native-audio-api/common/cpp/audioapi/core/Constants.h
+++ b/node_modules/react-native-audio-api/common/cpp/audioapi/core/Constants.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstddef>
 #include <cmath>
 #include <limits>
 
